packages/metascraper-title/index.js

```diff
@@ -13,7 +13,7 @@ module.exports = () => ({
     wrap($ => $('meta[property="og:title"]').attr('content')),
     wrap($ => $('meta[name="twitter:title"]').attr('content')),
     wrap($ => $('.post-title').text()),
-    wrap($ => $('.entry-title').text()),
+    wrap($ => ($('.entry-title').length === 1 ? $('.entry-title').text() : '')),
```

Comment:
You can use [`$filter`](https://github.com/microlinkhq/metascraper/blob/master/packages/metascraper-helpers/index.js#L105) helper instead to pick the first result with a non empty value